app/lib/custom_log_formatter.rb

```diff
@@ -56,4 +57,11 @@ def remove_post_params
   def method_is_post_or_put_or_patch?
     hash.dig(:payload, :method).in?(%w[PUT POST PATCH])
   end
+
+  def remove_application_details
+    return unless hash[:message].include?("RecruitsApi::ImportApplicationJob") && hash.dig(:payload, :arguments).present?
+
+    cleaned_arguments = JSON.parse(hash[:payload][:arguments]).first&.slice("id", "type")
```

Comment:
Is `hash[:payload][:arguments]` an `Array`?  The assignment to `cleaned_arguments` will be a `Hash`. Is this ok for the logger?